src/Components/MapView/MapView.jsx

```diff
@@ -17,8 +17,45 @@ function MapView(props) {
 
   const zoomBoundaries = { min: 12, max: 14 };
 
+  // const refContainer = useRef(null);
```

Comment:
Checking that the commented out code is intentional -- I saw a few console.logs were included too so just being sure

Reply:
@snschroeder is this intentional?